helm-charts/verticadb-operator/values.yaml

```diff
@@ -18,7 +18,14 @@
 
 
 image:
-  name: vertica/verticadb-operator:1.3.1
+  repo: null # null = dockerhub
+  name: vertica/verticadb-operator
+  tag: 1.3.1
+
+rbac_proxy_image:
+  repo: gcr.io
+  name: kubebuilder/kube-rbac-proxy
```

Comment:
Can you add this new parameter to the helm-charts/verticadb-operator/README.md file?